docs/production/upgrade-or-modify.md

```diff
@@ -363,6 +365,10 @@ instructions for other supported platforms.
    /home/zulip/deployments/current/manage.py audit_fts_indexes
    ```
 
+8. [Upgrade from Bionic to
```

Comment:
We should change step 6 above this to say "Latest Zulip 4.x release", right?

Reply:
Changed.